dev-requirements.txt

```diff
@@ -6,14 +6,13 @@ pytest-cov
 Mock
 
 #Linting
+enum34
```

Comment:
I'm fine having more requirements, but where do we use `enum34` and `configparser`?